### ISSUE TITLE
Styler breaks ranges at the start of pipes

### ISSUE THREAD

**Issue by cschmatzler** [adobe/elixir-styler#76]
## Versions

* Elixir: 1.15.5-otp-26
* Styler: 0.9.1

## Example Input

```elixir
start..23//step
|> Stream.drop_while(fn x -> x < hour end)
```

## Current Behaviour

Output:

```elixir
start
|> ..//(23, step)
|> Stream.drop_while(fn x -> x < hour end)
```

Styler
1. moves the end of the range into the step
2. moves the call to `..` into a pipe, which is invalid syntax with 
  ```
  unexpected token: "." (column 10, code point U+002E)
      |
   22 |       |> ..//(23, step)
  ```

**Comment by novaugust** [comment]
thanks for the issue! will have it fixed up in a jiffy

**Comment by novaugust** [comment]
fix published in v0.9.3, thanks again :)
